components/ui/sosial-media/report/modal-report-menu-sosial-media.tsx

```
@@ -0,0 +1,327 @@
+                        },
+                        {
+                            id: 7,
+                            text: "Animal violence"
+                        },
+                    ]
+                }
+
+                setDataReportTypeContent(valueListReportTypeConten)
+                setModalReportTypeContentSosialMedia(true)
+                break;
+            case 4:
```

Biome (2.1.2)

[error] Duplicate case label.

The first similar label is here:


(lint/suspicious/noDuplicateCase)